app/uk/gov/hmrc/tradergoodsprofilesrouter/service/MaintainProfileService.scala

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
I think we have to abstract out a service for the rest of the endpoints too if we are doing for one, so as to have some consistency. And they can all extend a BaseService where shared functionalities like `logMessageAndReturnError` can reside

Reply:
Yeah I agree, but I will do that as a separate PR, I want to keep this one a bit smaller

Reply:
Alright then. Happy to resolve.